### ISSUE TITLE
[Bug] Better exception handling 

### ISSUE THREAD

**Issue by jtaleric** [cloud-bulldozer/cpt-dashboard#37]
![image](https://github.com/cloud-bulldozer/ocp-perf-dashboard/assets/3965163/e5e0b982-00e1-4e8d-b16e-4c65adfae5a5)

This sometimes pops up on the main dashboard...

**Comment by jtaleric** [comment]
This seems to happen randomly if I leave the window open? Just happened again. 

![image](https://github.com/cloud-bulldozer/ocp-perf-dashboard/assets/3965163/45b2dce4-d73a-4234-9799-c17896658e2c)

**Comment by jtaleric** [comment]
cc @chentex / @athiruma

**Comment by chentex** [comment]
Hi @jtaleric, can you provide browser and version, I have tried in both I have and I have not hit the issue.

**Comment by jtaleric** [comment]
```
Google Chrome	117.0.5938.92 (Official Build) (64-bit) ￼
Revision	67649b10b92bb182fba357831ef7dd6a1baa5648-refs/branch-heads/5938_62@{#14}
OS	Linux
JavaScript	V8 11.7.439.16
```

**Comment by jtaleric** [comment]
Literally on page load I see 8 errors : 
![image](https://github.com/cloud-bulldozer/ocp-perf-dashboard/assets/3965163/4e41700a-ac92-415c-96a0-d540e62d5ab9)

**Comment by athiruma** [comment]
> Literally on page load I see 8 errors : ![image](https://user-images.githubusercontent.com/3965163/272565137-4e41700a-ac92-415c-96a0-d540e62d5ab9.png)

We can ignore this warnings. Some of the warnings are resolved in the pr [#35](https://github.com/cloud-bulldozer/ocp-perf-dashboard/pull/35)
